### DISCUSSION TITLE
看了你们一下午的文档，依然不知道怎么入手，你们就没有给出一个成功的例子

### DISCUSSION THREAD

**Discussion by b3601993** [simple-robot/Discussions#120] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

QQ频道

### 疑问描述

刚开始百度搜索，得到的都是你们版本2的例子，想着既然你们重构了，那就使用版本3。

结果你们的所有的例子中，就没有一个快速上手的例子，即使按照指示，也只是一个空壳。

这对于第一次玩QQ机器人的人来说，非常不友好。

官网文档的第一页，能不能给一个快速开始，只要修改账号和密码，就能成功的例子？

有个成功的例子，后续就可以自己打断点，了解怎么玩的。而不是要我们满世界查找资料，谢谢！

**Comment by ForteScarlet** [comment]
文档和示例对于我们来说的确一直都是痛点，但是想要完善需要非常庞大的时间与精力，而这都是我们非常匮乏的资源。
**`QQ频道组件`** 是近期刚刚经历了大型重构的，而且版本仍处于 `ALPHA` 版本，因此对于文档等内容更是无暇顾及。

目前simbot大多数应用都在 **`mirai组件`** 上，因此它也相对更清晰一些。现阶段如果你打算使用QQ频道组件，可以去 [QQ群](https://jq.qq.com/?_wv=1027&k=0HVo8aFV) 联系群主得到一定程度的帮助，也可以去 [QQ频道](https://pd.qq.com/s/anzubgojn) 通过 `@法欧莉斯卡雷特 今天的我` 来观察对于QQ频道组件的简单应用效果。

不过既然提到了，我们会近期尽可能先简单的补充一下相关的文档
